en, zh: refine some docs (#408)
pingcap/docs-dm: en/task-configuration-file-full.md

# Configuration arguments of the Loader processing unit.
loaders:
  global:                            # The configuration name of the processing unit.
    pool-size: 16                    # The number of threads that execute Mydumper SQL files concurrently in Loader (16 by default). When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load condition.

>     dir: "./dumped_data"             # The directory output by Mydumper that Loader reads ("./dumped_data" by default). Directories for different tasks of the same instance must be different (Mydumper outputs the SQL file based on the directory).

Reviewer: 原文的意思是“mydumper 输出 SQL 文件的地址，同时也是 loader 读取文件的地址”吗？loader 读取的是什么？

>     dir: "./dumped_data"             # The directory output by Mydumper that Loader reads ("./dumped_data" by default). Directories for different tasks of the same instance must be different (Mydumper outputs the SQL file based on the directory).

Reviewer 2: > 原文的意思是“mydumper 输出 SQL 文件的地址，同时也是 loader 读取文件的地址”吗？

是的

> loader 读取的是什么？

在这个路径下读取文件

and I prefer "directory" since its config name is "dir"

>     dir: "./dumped_data"             # The directory output by Mydumper that Loader reads ("./dumped_data" by default). Directories for different tasks of the same instance must be different (Mydumper outputs the SQL file based on the directory).

Reviewer 2: 所以中文的“地址”也可以改成“目录” 😂

>     dir: "./dumped_data"             # The directory output by Mydumper that Loader reads ("./dumped_data" by default). Directories for different tasks of the same instance must be different (Mydumper outputs the SQL file based on the directory).

Reviewer: 好，那就用 directories，把 addresses 去掉。可以帮忙把中文和英文都更新下～ @CharLotteiu

>     dir: "./dumped_data"             # The directory output by Mydumper that Loader reads ("./dumped_data" by default). Directories for different tasks of the same instance must be different (Mydumper outputs the SQL file based on the directory).

Author: 好的没问题。

# Configuration arguments of the Syncer processing unit.
syncers:
  global:                            # The configuration name of the processing unit.

>     worker-count: 16                 # The number of threads that replicate binlog events concurrently in Syncer. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.

Author: ```suggestion
    worker-count: 16                 # The number of threads that replicate binlog events concurrently in Syncer. When multiple instances are migrating data to TiDB at the same time, reduce the value according to the load.
```

>     worker-count: 16                 # The number of threads that replicate binlog events concurrently in Syncer. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the lo a d.

Reviewer: ```suggestion
    worker-count: 16                 # The number of threads that replicate binlog events concurrently in Syncer. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.
```

  -
    source-id: "mysql-replica-02"                   # The `source-id` in source.toml.

>     mydumper-thread: 4                              # The number of threads that Mydumper uses for dumping data. `mydumper-thread` corresponds to `threads` in the configuration of mydumper processing unit, while `mydumper-thread` has higher priority they are both configured.

Reviewer: ```suggestion
    mydumper-thread: 4                              # The number of threads that Mydumper uses for dumping data. `mydumper-thread` corresponds to the `threads` configuration of the mydumper processing unit. `mydumper-thread` has overriding priority when the two items are both configured.
```
Divide sentences with different sub-topics smartly.

>     loader-thread: 16                               # The number of threads that Loader uses for loading data. `loader-thread` corresponds to `pool-size` in the configuration of loader processing unit, while `loader-thread` has higher priority when they are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load condition.

Author: ```suggestion
    loader-thread: 16                               # The number of threads that Loader uses for loading data. `loader-thread` corresponds to the `pool-size` configuration of the loader processing unit. `loader-thread` has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.
```

>     loader-thread: 16                               # The number of threads that Loader uses for loading data. `loader-thread` corresponds to the `pool-size` configuration of the loader processing unit. `loader-thread` has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.

Author: ```suggestion
    loader-thread: 16                               # The number of threads that Loader uses for loading data. `loader-thread` corresponds to the `pool-size` configuration of the loader processing unit. `loader-thread` has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, reduce the value according to the load.
```

>     syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to `worker-count` in the configuration of the syncer processing unit, while `syncer-thread`  has higher priority when they are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load condition.

Reviewer: ditto

>     syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to `worker-count` in the configuration of the syncer processing unit, while `syncer-thread`  has higher priority when they are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load condition.

Author: ```suggestion
    syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to the `worker-count` configuration of the syncer processing unit. `syncer-thread`  has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.
```

>     syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to the `worker-count` configuration of the syncer processing unit. `syncer-thread`  has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.

Reviewer 2: "slightly" may indicate "don't change much", but perhaps user has a heavy workload when 2 instances, and need drop to half, for example. 😂 I prefer "properly" or something

>     syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to the `worker-count` configuration of the syncer processing unit. `syncer-thread`  has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.

Author: Get that, but can I just omit "适当" in the English version and keep it as "When multiple instances are migrating data to TiDB at the same time, reduce the value according to the load"? It seems like there's no need to emphasize the word "properly" because "according to the load" already tells the user what they should depend on.

>     syncer-thread: 16                               # The number of threads that Syncer uses for replicating incremental data. `syncer-thread` corresponds to the `worker-count` configuration of the syncer processing unit. `syncer-thread`  has overriding priority when the two items are both configured. When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.

Reviewer 2: OK
